src/app/consumption-estimation/consumption-estimation.component.html

```diff
@@ -163,61 +139,37 @@ <h2 class="form-header">Meter</h2>
                         <mat-radio-group formControlName="type" class="flex place-content-evenly">
                             <div>
                                 <div><h3><strong>Meter no.1</strong></h3></div>
-                                <mat-radio-button [ngClass]="{ 'is-invalid': meterType_step && meterType['type'].errors }" value="manualMeter" class="block px-3.5">Manual Meter</mat-radio-button>
-                                <mat-radio-button [ngClass]="{ 'is-invalid': meterType_step && meterType['type'].errors }" value="smartMeter" class="block px-3.5">Smart Meter</mat-radio-button>
+                                <mat-radio-button [ngClass]="{ 'is-invalid': meterType_step && meterType['type'].errors }" value="Manual" class="block px-3.5">Manual Meter</mat-radio-button>
+                                <mat-radio-button [ngClass]="{ 'is-invalid': meterType_step && meterType['type'].errors }" value="Smart" class="block px-3.5">Smart Meter</mat-radio-button>
```

Comment:
dont use hardcoded values here

Reply:
Also lots of repetition, dont do it for time but this should be a seperate component etc